carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -162,6 +162,11 @@ std::shared_ptr<Geofence> WMBroadcaster::geofenceFromMsg(const cav_msgs::Traffic
     addPassingControlLineFromMsg(gf_ptr, msg_v01, affected_llts);
   }
 
+ if (msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED_CHOICE || msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED)
```

Comment:
msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED would not happen I think

Reply:
it should check if 
msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED_CHOICE && msg_detail.closed==cav_msgs::TrafficControlDetail::CLOSED